test/files/neg/t6124.check

```diff
@@ -1,28 +1,49 @@
-t6124.scala:3: error: trailing separator is not allowed
+t6124.scala:3: error: illegal separator
   def i: Int = 123_456_
                       ^
-t6124.scala:4: error: trailing separator is not allowed
+t6124.scala:4: error: illegal separator
   def j: Long = 123_456_L * 1000
                        ^
```

Comment:
I realise this isn't changing here, but there's some argument to be made that you should be able to put a separator in front of the type identifier (or whatever it's called)

Reply:
That would be some argument. Maybe the Long interpoLator could support that. The folks who use it like to say, Lator, alligator.